src/Cameras/babylon.camera.ts

```diff
@@ -413,8 +414,15 @@
             return this._computedViewMatrix;
         }
 
+        public freezeProjectionMatrix(projection?: Matrix): void {
+            this._doNotComputeProjectionMatrix = true;
+            if (projection !== undefined) {
+                this._projectionMatrix = projection;
+            }
+        };
+        
         public getProjectionMatrix(force?: boolean): Matrix {
-            if (!force && this._isSynchronizedProjectionMatrix()) {
+            if ((!force && this._isSynchronizedProjectionMatrix()) || this._doNotComputeProjectionMatrix) {

```

Comment:
Oooh, so sorry... I could have done it... 
C'était grand w-e férié + pont là, j'étais en déplacement familial, sans ordi ni internet pour souffler un peu... :-p